api/api.go

```diff
@@ -112,7 +112,9 @@ func readObjectCache(in io.Reader) (*StaticData, error) {
 				if i == "define hostgroup {" || strings.TrimSpace(i) == "}" || i == "" {
 					// Ignore these lines
 				} else {
-					thisgroup[strings.TrimSpace(strings.Fields(i)[0])] = strings.TrimSpace(strings.Fields(i)[1])
+					if(len(strings.Fields(i)) != 1) {
```

Comment:
if(len(strings.Fields(i)) != 1)  shouldnt this be != 0 ? ... not sure i understand why !=1.

Reply:
I merged the pull request locally, and @sulochan you are right, it should be !=0

This fix is important, it fixes the "index out of range" bug for some objects.cache.
@bose1993, can you fix it, so that it can be merged ? :)

Reply:
yeah i am happy to merge once thats fixed. Also, this patch has been here for a while so not sure if @bose1993 is still working on this. I am happy to merge another PR if someone wants to open a new patch for this in case @bose1993 does not respond in the next fews days.